be/src/storage/replication_utils.cpp

```diff
@@ -277,9 +280,9 @@ Status ReplicationUtils::download_remote_snapshot(
         }
 
         total_file_size += file_size;
-        uint64_t estimate_timeout_sec = file_size / config::download_low_speed_limit_kbps / 1024;
-        if (estimate_timeout_sec < config::download_low_speed_time) {
-            estimate_timeout_sec = config::download_low_speed_time;
+        uint64_t estimate_timeout_sec = file_size / config::replication_min_speed_limit_kbps / 1024;
```

Comment:
is there configvar validation? try to avoid divide-0 error.

Reply:
no, I will add